### PR TITLE
more-sharing-buttons

### PR DESCRIPTION
**Pull request by py-den**
__What issue are you solving? 

i have added another sharing buttons for [vk](https://vk.com) and [facebook](https://facebook.com)

### CODE REVIEW

File: congratulations/Congratulations.py [hunk 1]
```diff
@@ -33,13 +33,32 @@ def __init__(self):
...
+
+        twitter_btn.pack(side=Tk.RIGHT)
+        facebook_btn.pack(side=Tk.RIGHT)
+        vk_btn.pack(side=Tk.RIGHT)
 
         # Tkinter instantiation
         self.root.mainloop()
     
+    def open_facebook_browser(self):
+        msg = quote("I just completed my " + str(self.challenge_duration) + "-days challenge! "+ self.challenge_hashtag)
```
Comment (lineaba) [inline congratulations/Congratulations.py]: Everytime a new function is added, a docstring is needed as documentation. This ensures that code is understandable for new contributors, and for us self in the future. You can take a look at the docstring in the open_twitter_browser(self) below for inspiration.

File: congratulations/Congratulations.py [hunk 2]
```diff
@@ -33,13 +33,32 @@ def __init__(self):
...
+        print(self.challenge_hashtag)
+        print(msg)
+        webbrowser.open_new_tab("https://www.facebook.com/sharer/sharer.php?u=https://github.com/prog-o-meter/prog-o-meter&quote=" + msg)
+
+    def open_vk_browser(self):
```
Comment (lineaba) [inline congratulations/Congratulations.py]: Everytime a new function is added, a docstring is needed as documentation. This ensures that code is understandable for new contributors, and for us self in the future. You can take a look at the docstring in the open_twitter_browser(self) below for inspiration.

**Comment by lineaba** [comment]
design
Here is my suggestion for the layout of the buttons. Let me know what you think

**Comment by py-den** [comment]
i changed layout to make it look like the picture, but there is a problem with resizing this window, probably i will later try to make the same design with other layout so that it will be resizing correctly

**Comment by lineaba** [comment]
Hey, I have merged your code, thanks for adding the buttons. It would be nice when you look at the redesign, if we could make sure the buttons are in the middle of the window. Right now they are slightly more to the left than the middle. I think this might be doable if you don't set a relx, because I think default is 0.5 (middle), but if not, try seeing what happens if you set the relx to 0.5. 
If you are not interested in working on that, that's ok, I'll open it as an issue for someone else to work on :)

**Pull request merged**